src/themes/greyVest.js

```diff
@@ -796,6 +814,17 @@ let TagComponent = observer(({ value, removeTag, tagStyle, onClick }) => (
 ))
 TagComponent.displayName = 'TagComponent'
 
+let WrappedDatePicker = observer(({ value, onChange, calendarType = 'US' }) => (
+  <DatePicker
+    value={value ? new Date(value) : null}
+    onChange={date =>
+      onChange({ target: { value: moment(date).format('YYYY-MM-DD') } })
```

Comment:
actually it seems like we should go the other way - instead of assuming onChange takes e.target.value, we could change the default html5 date input to just pass e.target.value in the onChange